.env

```diff
@@ -9,3 +9,4 @@ APP_SECRET=a1956d4d5d173273db7d638ed5b50ca0
 
 JOB_QUEUE_TOKEN=sampleToken
 TEST_INTERNAL_API_APPLICATION_TOKEN=
+STORAGE_API_URL=https://connection.keboola.com/
```

Comment:
just curious, is this required for this change?

Reply:
no, it was just missing for running tests locally